test_filter_expired_tmp_jobs is failing on master

```FAILURES
test_filter_expired_tmp_jobs

mock_get_temporary_jobs = <MagicMock name='get_temporary_jobs_for_service_instance' id='73619536'>

    @mock.patch('paasta_tools.cleanup_chronos_jobs.chronos_tools.get_temporary_jobs_for_service_instance')
    def test_filter_expired_tmp_jobs(mock_get_temporary_jobs):
        two_days_ago = datetime.datetime.now() - datetime.timedelta(days=2)
        one_hour_ago = datetime.datetime.now() - datetime.timedelta(hours=1)
        mock_get_temporary_jobs.side_effect = [
                                              [{'name': 'tmp foo bar',
                                                'lastSuccess': two_days_ago.isoformat()}],
                                              [{'name': 'tmp anotherservice anotherinstance',
                                                'lastSuccess': one_hour_ago.isoformat()}]]
>       actual = cleanup_chronos_jobs.filter_expired_tmp_jobs(mock.Mock(), ['foo bar', 'anotherservice anotherinstance'])

tests/test_cleanup_chronos_jobs.py:61:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

client = <Mock id='73624464'>, job_names = ['foo bar', 'anotherservice anotherinstance']

    def filter_expired_tmp_jobs(client, job_names):
        """
        Given a list of temporary jobs, find those ready to be removed. Their
        suitablity for removal is defined by two things:

            - the job has completed (irrespective of whether it was a success or
              failure)
            - the job completed more than 24 hours ago
        """
        expired = []
        for job_name in job_names:
            service, instance = chronos_tools.decompose_job_id(job_name)
            temporary_jobs = chronos_tools.get_temporary_jobs_for_service_instance(
                client=client,
                service=service,
                instance=instance
            )
            for job in temporary_jobs:
                last_run_time, last_run_state = chronos_tools.get_status_last_run(job)
                if last_run_state != chronos_tools.LastRunState.NotRun:
                    if ((datetime.datetime.now(dateutil.tz.tzutc()) -
>                        dateutil.parser.parse(last_run_time)) >
                            datetime.timedelta(days=1)):
E                           TypeError: can't subtract offset-naive and offset-aware datetimes

paasta_tools/cleanup_chronos_jobs.py:135: TypeError
```

--- a/tests/test_cleanup_chronos_jobs.py
+++ b/tests/test_cleanup_chronos_jobs.py
@@ -13,6 +13,7 @@
 # limitations under the License.
 import datetime
 
+import dateutil
 import mock
 
 from paasta_tools import cleanup_chronos_jobs
@@ -51,8 +52,8 @@ def test_deployed_job_names():
 
 @mock.patch('paasta_tools.cleanup_chronos_jobs.chronos_tools.get_temporary_jobs_for_service_instance')
 def test_filter_expired_tmp_jobs(mock_get_temporary_jobs):
-    two_days_ago = datetime.datetime.now() - datetime.timedelta(days=2)
-    one_hour_ago = datetime.datetime.now() - datetime.timedelta(hours=1)
+    two_days_ago = datetime.datetime.now(dateutil.tz.tzutc()) - datetime.timedelta(days=2)
+    one_hour_ago = datetime.datetime.now(dateutil.tz.tzutc()) - datetime.timedelta(hours=1)
     mock_get_temporary_jobs.side_effect = [
                                           [{'name': 'tmp foo bar',
                                             'lastSuccess': two_days_ago.isoformat()}],

